x-pack/plugins/ml/public/application/services/ml_api_service/trained_models.ts

```diff
@@ -109,6 +110,50 @@ export function trainedModelsApiProvider(httpService: HttpService) {
       });
     },
 
+    /**
+     * Fetches all ingest pipelines
+     *
```

Comment:
Guess we can remove all the comment lines with just `*` in this file.

Reply:
Updated in 4b1791c5523f98483b24f0ca0dd3f1e88b855a50